### DISCUSSION TITLE
Problem with displaying the latest entries in Hunt

### DISCUSSION THREAD

**Discussion by tswiderski1** [Security-Onion-Solutions/securityonion#8563] [General] [answered]
If I have selected multiple entries from many days to display in the HUNT tab, it does not show me the latest entries. When I change the filter to 1 hour, I see these latest entries.

There is no such problem in Kiban.
![1hour](https://user-images.githubusercontent.com/111741583/185892502-e2224dd8-2720-415f-91f1-12d302b22641.png)
![15days](https://user-images.githubusercontent.com/111741583/185892506-3b2b6054-9692-43d2-8afa-74add05c9425.png)

**Comment by dougburks** [comment, marked as answer]
You are exceeding the `Fetch Limit` of `5000` events and so the most recent events are not returned. 

Here are a couple of options:
- filter your query until the `Total Found` is less than the `Fetch Limit`
- add ` | sortby @timestamp` to your query

For more information, please see https://docs.securityonion.net/en/2.3/dashboards.html.
